src/AutoRest.CSharp/Common/Input/CodeModelPartials.cs

```diff
@@ -159,6 +159,8 @@ internal static class XMsFormat
         public const string DurationConstant = "duration-constant";
         public const string ETag = "etag";
         public const string ResourceType = "resource-type";
+        public const string Object = "object";
```

Comment:
Please also update the doc here https://github.com/Azure/autorest.csharp/blob/feature/v3/docs/mgmt/polishing.md#change-format-by-name-rules

Reply:
Thanks, updated.